docs/docs/sample-scripts/spo/list-all-sites-with-app-installed/assets/sample.json

```diff
@@ -0,0 +1,50 @@
+[
+  {
+    "name": "pnp-list-all-sites-with-app-installed",
+    "source": "pnp",
+    "title": "List SharePoint sites where the particular app is installed",
+    "url": "https://pnp.github.io/cli-microsoft365/sample-scripts/spo/list-all-sites-with-app-installed",
+    "creationDateTime": "2022-02-19",
+    "updateDateTime": "2023-02-28",
+    "shortDescription": "This script will retrieve all sites in SharePoint where a particular app is installed.",
+    "longDescription": [
+      "This script will retrieve all sites in SharePoint where a particular app is installed."
+    ],
+    "products": [
+      "SharePoint"
+    ],
+    "categories": [],
+    "tags": [
+       "reports",
+       "sites"
+    ],
+    "metadata": [
+      {
+        "key": "CLI-FOR-MICROSOFT365",
+        "value": "6.4.0"
+      }
+    ],
+    "thumbnails": [
+      {
+        "type": "image",
+        "order": 100,
+        "url": "https://raw.githubusercontent.com/pnp/cli-microsoft365/main/docs/docs/sample-scripts/spo/list-all-checked-out-files/assets/preview.png",
```

Comment:
we should update this url to take the preview image from current sample, not different one. So it should be something like 
```suggestion
        "url": "https://raw.githubusercontent.com/pnp/cli-microsoft365/main/docs/docs/sample-scripts/spo/list-all-sites-with-app-installed/assets/preview.png",
```